enctests/testframework/main.py

```diff
@@ -397,6 +404,10 @@ def main():
     # Serialize to *.otio
     otio.adapters.write_to_file(timeline, args.output)
 
+    # Generate any reports (if specified in file)
```

Comment:
Should this be triggered/toggled by an argument? `--generate-html` for instance?

Reply:
I can do the reverse, add a skip-reports flag, it feels like if you have specified the report generation in the file, it should run it by default.

Reply:
agree. opt-out rather than opt-in sounds good